cmd/libs/go2idl/client-gen/generators/generator_for_group.go

```diff
@@ -103,7 +108,11 @@ func (g *genGroup) GenerateType(c *generator.Context, t *types.Type, w io.Writer
 	sw.Do(newClientForConfigTemplate, m)
 	sw.Do(newClientForConfigOrDieTemplate, m)
 	sw.Do(newClientForRESTClientTemplate, m)
-	sw.Do(setClientDefaultsTemplate, m)
+	if g.version == "unversioned" {
```

Comment:
I think "unversioned" and "__internal" are mixed here. Is there a TODO to clarify this? If not can you add one somewhere?